src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -44,9 +44,9 @@ public class ProviderInfo
         public string Name { get; }
```

Comment:
@iSazonov, your last commit had 75 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

help can be found for AllUsers Scope.Error occurred in Describe block
```powershell
Failed to update Help for the module(s) :
'PackageManagement'
The Update-Help command failed because the specified module does not support updatable help. Use Get-Help -Online or look online for help for the commands in this module.
At D:\a\1\s\test\powershell\engine\Help\HelpSystem.Tests.ps1:31 char:5
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'PackageManagement' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'PackageManagement' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules for user scope..Validate Update-Help for module 'PackageManagement' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell modules for user scope..Validate Update-Help for module 'PackageManagement' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```